packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -38,7 +39,7 @@ const Breakouts = WebexPlugin.extend({
     mainLocusUrl: 'string', // the locus url of the main session
     groups: 'array', // appears when create breakouts
   },
-
+  isCallPreassignments: false, // Controlling the lifecycle of the pre-assign API
```

Comment:
issue: I think you misunderstood the request here. It is better to put the attributes of a breakouts object in the props so they can be get/set correctly, rather than using a flag on the class itself i.e.
```ts
    mainLocusUrl: 'string', // the locus url of the main session
    groups: 'array', // appears when create breakouts
    shouldFetchPreassignments: 'boolean', // Controlling the lifecycle of the pre-assign API
```

then later
```ts
this.set('shouldFetchPreassignments', true);
```

Reply:
Yes, I lost your mention before, let's follow this rule which object in the props so they can be get/set correctly.